ldk-server/src/util/config.rs

```diff
@@ -61,6 +63,11 @@ impl TryFrom<TomlConfig> for Config {
 			(rabbitmq.connection_string, rabbitmq.exchange_name)
 		};
 
+		#[cfg(not(feature = "experimental-lsps2-support"))]
+		let lsps2_service_config: Option<LSPS2ServiceConfig> = None;
+		#[cfg(feature = "experimental-lsps2-support")]
+		let lsps2_service_config = Some(toml_config.liquidity.expect("`liquidity.lsps2_server` must be defined in config if enabling `experimental-lsps2-support` feature.").lsps2_service.into());
```

Comment:
Why do we `expect` here but return an `Err` if `rabbitmq` config is not properly set?